src/web/event-notifications/event-notification-types/SlackNotificationSummary.jsx

```diff
@@ -1,52 +1,55 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-
 import CommonNotificationSummary from 'components/event-notifications/event-notification-types/CommonNotificationSummary';
 
 function SlackNotificationSummary({ notification, ...restProps }) {
   return (
     <CommonNotificationSummary {...restProps} notification={notification}>
-        <tr>
-          <td>Color</td>
-          <td>{notification?.config?.color}</td>
-        </tr>
-        <tr>
-          <td>Webhook URL</td>
-          <td>{notification.config.webhook_url}</td>
-        </tr>
-        <tr>
-          <td>Channel</td>
-          <td>{notification.config.channel}</td>
-        </tr>
-        <tr>
-          <td>Custom Message</td>
-          <td>{notification.config.custom_message}</td>
-        </tr>
-        <tr>
-          <td>User Name</td>
-          <td>{notification.config.user_name}</td>
-        </tr>
-        <tr>
-          <td>Notify Channel</td>
-          <td>{notification.config.notify_channel ? 'Yes' : 'No'}</td>
-        </tr>
-        <tr>
-          <td>Link Names</td>
-          <td>{notification.config.link_names ? 'Yes' : 'No'}</td>
-        </tr>
-        <tr>
-          <td>Icon URL</td>
-          <td>{notification.config.icon_url}</td>
-        </tr>
-        <tr>
-          <td>Icon Emoji</td>
-          <td>{notification.config.icon_emoji}</td>
-        </tr>
-        <tr>
-          <td>Graylog URL</td>
-          <td>{notification.config.graylog_url}</td>
-        </tr>
-      </CommonNotificationSummary>
+      <tr>
+        <td>Color</td>
+        <td>{notification?.config?.color}</td>
+      </tr>
+      <tr>
+        <td>Webhook URL</td>
+        <td>{notification.config.webhook_url}</td>
+      </tr>
+      <tr>
+        <td>Channel</td>
+        <td>{notification.config.channel}</td>
+      </tr>
+      <tr>
+        <td>Custom Message</td>
+        <td>{notification.config.custom_message}</td>
+      </tr>
+      <tr>
+        <td>Backlog Message Size</td>
```

Comment:
Again, we should be consistent with the labeling for this value across all three pages.

Reply:
yeah agree. the original story uses multiple conventions and they both mean the same,

https://github.com/Graylog2/graylog-plugin-integrations/issues/623